applications/app/views/fragments/interactiveBody.scala.html

```diff
@@ -1,32 +1,25 @@
 @(page: InteractivePage)(implicit request: RequestHeader)
 @import common.Edition
-@import views.support.{RenderClasses, InteractiveCleaner}
+@import views.support.RenderClasses
 @import views.support.TrailCssClasses.toneClass
-@import conf.switches.Switches.interactivePressing
+@import views.InteractiveBodyCleaner
 @import views.support.Commercial._
 
 @body(page.interactive, page.item.content.isImmersive, isPaidContent(page.interactive, page))
 
 @bodyRaw(interactive: model.Interactive) = {
     @interactive.maybeBody.map { body =>
-        @HtmlFormat.raw(body)
+        @InteractiveBodyCleaner(interactive)
     }.getOrElse {
+        @* legacy support? *@
```

Comment:
I tried to track down if this was ever used. I can't see any scenario where an interactive made now would have `maybeBody == None` so was hoping to clean it up. But yeah, still a question!